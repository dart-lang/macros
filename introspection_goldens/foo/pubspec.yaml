name: foo
publish-to: none
resolution: workspace

environment:
  sdk: ^3.6.0-114.0.dev

dependencies:
  _test_macros: any
