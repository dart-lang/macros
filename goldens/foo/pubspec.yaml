name: foo
publish-to: none
resolution: workspace

environment:
  sdk: ^3.7.0-39.0.dev

dependencies:
  _test_macros: any
